osu.Game/OsuGameBase.cs

```diff
@@ -110,6 +110,10 @@ private void load()
             dependencies.CacheAs(this);
             dependencies.Cache(LocalConfig);
 
+            BindableVisualSettings visualSettings;
+            AddInternal(visualSettings = new BindableVisualSettings());
+            dependencies.Cache(visualSettings);
```

Comment:
I really don't like visual settings being global like this, and especially not at an OsuGameBase level.